### PR TITLE
obsolete MONDO:0800297

### LINKED ISSUE #5922: neuronal ceroid lipofuscinosis, late infantile [Merge]
**Mondo term (ID and Label)**
MONDO:0800297 neuronal ceroid lipofuscinosis, late infantile

**Reason for deprecation**
Seems to be redundant (at least in name) with MONDO:0015674  late infantile neuronal ceroid lipofuscinosis
However, the Orphanet x-ref on this is ORPHA:168491 which points to multiple OMIM records; So it seems the Orphanet concept is all CLNs that have 'late infantile' onset regardless of genotype; and it's unclear to me if the MIM 'included' concept 'LINCL' is supposed to be connecting the former classification of "CLNs by age of onset" to the "CLNs by gene" newer classification approach. https://omim.org/entry/204500 
>The CLNs were originally classified broadly according to age at onset, with CLN2 as the late infantile-onset (LINCL) form with onset between 2 and 4 years of age. With the identification of molecular defects, however, the CLNs are now classified numerically according to the underlying gene defect. CLN2 refers to CLN caused by mutation in the CLN2 gene, regardless of the age at onset.

**Term to be merged with**
MONDO:0015674  late infantile neuronal ceroid lipofuscinosis

**Your nano-attribution (ORCID)**
If you don't have an ORCID, you can sign up for one [here](https://orcid.org/)

### PR DESCRIPTION
**Pull request by sabrinatoro**
Closes #5922

### CODE REVIEW

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -518772,17 +518772,15 @@ property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/
...
 [Term]
 id: MONDO:0800297
-name: neuronal ceroid lipofuscinosis, late infantile
-comment: Reason: duplicate. This will be merged with MONDO:0015674 late infantile neuronal ceroid lipofuscinosis
+name: obsolete neuronal ceroid lipofuscinosis, late infantile
 subset: inferred_rare
-subset: obsoletion_candidate
 subset: rare
-synonym: "LINCL" EXACT [OMIM:204500]
 xref: OMIM:204500 {source="MONDO:includedEntryInOMIM"}
```
[the remark this answers is not in the document]
Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:518778]: @sabrinatoro could you please clarify what happened here? I don't see included in entries on this OMIM record, was it previously included on this OMIM record, then OMIM obsoleted it?

[the remark this answers is not in the document]
Comment (sabrinatoro) [reply src/ontology/mondo-edit.obo:518778]: yes, it was included in OMIM, and OMIM removed/obsoleted this "included" record.

[the remark this answers is not in the document]
Comment (matentzn) [reply src/ontology/mondo-edit.obo:518778]: I would just drop this, no need to keep a record of this; but document this somewhere ("if the record was previously marked as included by OMIM and is now not anymore, we remove the assertion")

[the remark this answers is not in the document]
Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:518778]: I agree!

and thanks for clarifying @sabrinatoro!

**Review by nicolevasilevsky — APPROVED** [review]

**Pull request merged**